webdriver/support/fixtures.py

```diff
@@ -115,3 +117,83 @@ def inner(path, query="", fragment=""):
                                   fragment))
         return rv
     return inner
+
+def switch_to_inactive(session):
```

Comment:
I don’t think this is a particularly obvious name. It tries to do a lot of things. Maybe it would be better to do some of these things in the test.